.github/workflows/docs.yml

```diff
@@ -0,0 +1,47 @@
+
+name: Documentation
+on:
+  push:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        
+      - name: Prepare apt repositories
+        run: | 
+          sudo sed -i '/deb-src/s/^# //' /etc/apt/sources.list
+          sudo apt update
+
+      - name: Install sugar-toolkit-gtk3
+        run: |
+          # enable deb-src repositories
```

Comment:
This comment seems to belong with this line:
  sudo sed -i '/deb-src/s/^# //' /etc/apt/sources.list

Reply:
yea, will remove it.

Reply:
Fixed it in https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/455/commits/81e29eee2fbf885c5464fec9238f609e64b1d388, the comment was redundant